system_tests/multi_writer_fallback_test.go

```diff
@@ -11,14 +11,273 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
 
 	"github.com/offchainlabs/nitro/arbnode"
 	"github.com/offchainlabs/nitro/cmd/chaininfo"
 	"github.com/offchainlabs/nitro/daprovider"
 	"github.com/offchainlabs/nitro/daprovider/das"
+	"github.com/offchainlabs/nitro/daprovider/referenceda"
+	"github.com/offchainlabs/nitro/util/signature"
 )
 
+// TestMultiWriterFailure_CustomDAShutdownWithAnyTrustAvailable tests that batch posting fails when CustomDA shuts down
+// without explicit fallback signal. This verifies the new behavior where DA errors must be
+// explicitly signaled via ErrFallbackRequested to trigger fallback. Even though AnyTrust is configured
+// and available, it should not be used without an explicit fallback signal.
```

Comment:
I'm just wondering - some of these tests do not fit into this PR, but rather to the #3949, right?